test/mpi/.gitignore

```diff
@@ -1061,6 +1061,7 @@
 /info/infoorder
 /info/infotest
 /info/infovallen
+/info/infowoinit
```

Comment:
Terrible name :) How do you spell out the name?

Reply:
Since other tests use underscore, I will rename this to `info_no_init`.

Reply:
In the latest commit, this test is `info_no_init`.